### PR TITLE
feat: added avatar component styles

### PR DESCRIPTION
**Pull request by aatishpandey**
image
Avatar component includes : 
- Avatar 
- Avatar with border
- Avatar with different sizes

### CODE REVIEW

File: components/component-stylesheets/avatar.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.avatar{
+    border-radius: 50%;
+    width: 8rem;
+    height: 8rem;
```
Comment (ttahm3d) [inline components/component-stylesheets/avatar.css]: offload the width and height to another class.

use combination of classes `.avatar .avatar-s` for - small avatar.

**Review by ttahm3d — COMMENTED** [review]
Split classes. rest all looks good.

**Review by SimranjitKaurMaan — COMMENTED** [review]
Please address review comments. Rest all looks good.

File: components/component-stylesheets/avatar.css (hunk 1 above)
Comment (aatishpandey) [reply components/component-stylesheets/avatar.css]: changes made in the recent commit.

**Pull request merged**